### ISSUE TITLE
Clean up branches

### ISSUE THREAD

**Issue by DieterHolvoet** [wieni/wmcontroller#58]
## Description

- [ ] Clean up unused branches (check Sourcegraph to make sure the branch is not directly required in a composer.json file)
- [ ] Switch default branch to `main`

**Comment by RobinHoutevelts** [comment]
Only `feature/split` is currently being used in some of our projects. But as I understand it, the `split-pre` is needed for a smooth upgrade?

These are all current branches:

- release/v0.6
- feature/split
- feature/split-pre

**Comment by DieterHolvoet** [comment]
Yes, those split branches will be needed for the upgrade. It's all documented [here](https://github.com/wieni/wmcontroller/blob/feature/split/UPGRADING.md). We should finish that upgrade some time soon, it's basically finished anyway.

`release/v0.6` is a branch for a certain version, so we probably shouldn't delete that one. The default branch is switched to main, so I think this issue is okay now.

**Comment by DieterHolvoet** [comment]
The `feature/split` and `feature/split-pre` branches can be deleted if they are not required anymore in any of your projects. I won't delete them.
